src/custom/state/orders/utils.ts

```diff
@@ -216,9 +216,6 @@ export function getEstimatedExecutionPrice(
   fee: string
 ): Price<Currency, Currency> {
   // TODO: implement estimation for partially fillable orders
```

Comment:
Is the TODO still relevant?

Reply:
Yes, because the calculation is still based on fill or kills
It's on the list of next tasks to deal with this